yarn.lock

```diff
@@ -17,6 +17,14 @@
   version "1.0.1"
   resolved "https://registry.yarnpkg.com/@hmcts/cmc-supported-browsers/-/cmc-supported-browsers-1.0.1.tgz#201d415b7efdf858c77ddccb66bbe9af258c03bc"
 
+"@hmcts/cmc-validators@^0.1.0":
+  version "0.1.0"
+  resolved "https://registry.yarnpkg.com/@hmcts/cmc-validators/-/cmc-validators-0.1.0.tgz#482d6264637d656368f6b209c3026ec6e29c85bc"
+  dependencies:
+    class-validator "^0.8.1"
+    ts-node "^3.3.0"
```

Comment:
I do not like the fact `ts-node` and `typescript` make the application bundle. We should remove these from dependencies of `@hmcts/cmc-validators` and make them `devDependencies`

Reply:
will update `cmc-validators`